src/fsharp/ScriptClosure.fs

```diff
@@ -43,6 +43,9 @@ type LoadClosure =
       /// The resolved pacakge references along with the ranges of the #r positions in each file.
       PackageReferences: (range * string list)[]
 
+      /// Whether we're decided to use .NET Framework analysis for this script
+      UseDotNetFramework: bool
```

Comment:
I suspect i've mention that UseDesktopFramework might be a better name, since coreclr is also the dotnet framework.

Reply:
Ah ok.  yup I was only dimly aware that the .NET Framework name have been resurrect